utils/content.ts

```diff
@@ -161,24 +193,24 @@ export const getContentList = (contentType: IContentType) => {
       throw new Error('You have to provide a content type');
   }
 
-  const content: { date?: Date; previewImage: any; id: string }[] = contentFiles
+  const content = contentFiles
     .filter((contentFile) => contentFile.endsWith('.md'))
-    .map((contentFile) => {
-      const filePath = `${contentDir}/${contentFile}`;
-      const rawContent = fs.readFileSync(filePath, {
+    .map((contentItem) => {
+      const contentPath = `${contentDir}/${contentItem}`;
+      const rawContent = fs.readFileSync(contentPath, {
         encoding: 'utf-8',
       });
 
       const { data } = matter(rawContent);
 
       return {
         ...data,
-        previewImage: data.previewImage || '/images/image-placeholder.png',
+        previewImage: data.previewImage ?? '/images/article-preview.png',
         id: uuid(),
-      };
+      } as IContent;
```

Comment:
Added type assertion because for TS the type returned had `previewImage` and `id` but this was wrong because  in lines 207-209 we spread the `data` object and added `previewImage` and `id`

Reply:
hmm, let's try to avoid a type assertion if possible